CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and `a-btn__icon-on-right` classes referenced in docs to correct
 `a-btn_icon__on-left` and `a-btn_icon__on-right` classes.
 - **cf-forms:** [PATCH] Changed label heading from 5 to 4, per design manual.
 - **cf-typography:** [PATCH] Remove bottom margin from `m-slug-header`.
+- **capital-framework:** Updates `gulp-autoprefixer` to `4.0.0`.
+- **capital-framework:** Updates `gulp-uglify` to `3.0.0`.
```

Comment:
It doesn't need it but to avoid confusion I vote we agree to *always* add one.

Under the hood, if `capital-framework` doesn't have a semver version (major, minor, patch) specified it'll use the component's semver version.

It's possible the bot will erroneously double increment `capital-framework` because we're listing it twice. It's okay if that happens, we can fix it and write a regression test. It's good to push the limits of the bot.